src/views/domain-page/domain-page-layout.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { Suspense } from 'react';
```

Comment:
I think you forgot to rename this to exclude  -layout

Reply:
I import this file in src/app/(Home)/domains/[domain]/[cluster]/layout.tsx so I left the "layout" name as is. Open to removing it though

Reply:
@adhityamamallan yeah i understand the reasoning behind it, but it seems that this would be the normal way of working on pages (they can be Layouts) so let's name them as they are perceived better than where they are used.